api/integrations/asana/zulip_asana_mirror

```diff
@@ -93,21 +97,24 @@ def send_zulip(topic, content):
     return client.send_message(message)
 
 def datestring_to_datetime(datestring):
+    # type: (str) -> datetime
     """
     Given an ISO 8601 datestring, return the corresponding datetime object.
     """
     return dateutil.parser.parse(datestring).replace(
-        tzinfo=dateutil.tz.gettz('Z'))
+        tzinfo=gettz('Z'))
 
 class TaskDict(dict):
     """
     A helper class to turn a dictionary with task information into an
     object where each of the keys is an attribute for easy access.
     """
     def __getattr__(self, field):
+        # type: (TaskDict, str) -> Any
```

Comment:
You usually don't need to annotate self; it's implied, so I'm guessing this will give an error.